modules/utils/subxt/Cargo.toml

```
@@ -23,10 +27,10 @@ features = ["sp-crypto-hashing"]
 
 [features]
 default = ["std"]
-wasm = ["subxt/web", "subxt/jsonrpsee", "reconnecting-jsonrpsee-ws-client/web"]
+wasm = ["subxt/web", "subxt/jsonrpsee", "reconnecting-jsonrpsee-ws-client/web", "subxt/reconnecting-rpc-client"]
```

We can remove `reconnecting-jsonrpsee-ws-client` dependency from the workspace since it's no longer used.
```suggestion
wasm = ["subxt/web", "subxt/jsonrpsee", "subxt/reconnecting-rpc-client"]
```

---

Okay